docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -301,6 +301,23 @@ netstat -nat | grep :8220
 Any response data indicates that the port is in use. This could be correct or not
 if you had intended to uninstall the {fleet-server}. In which case, re-check and continue.
 
+[discrete]
+[[fleet-agent-stuck-on-updating]]
+==  The {agent} is stuck in status `updating`
+
+The {agengt} can hung up in some rare situations in the `updating` status. This can be cleaned as authenticated `superuser`.
+After this the {agent} will return the real status again.
```

Comment:
This `force` unenrolls the elastic agent. It will not show up again. This command is for elastic-agent that do not exist anymore.

Reply:
This description is not clear. What rare situations cause this problem?  If you force unenroll and revoke tokens, how can the agent return the real status?

If this section is meant to document how to force unenrollment, can we tweak [the section we already have](https://www.elastic.co/guide/en/fleet/current/fleet-troubleshooting.html#agent-hangs-while-unenrolling) to cover the agent hanging at `updating` (rather than add a new section)?

Reply:
@dedemorton You are correct here force unenrolling an agent will not allow an agent to return to the real status it will clean old agents stuck in a weird status.

Reply:
thx @nchaulet for the clarification. I'll change the wording.

Reply:
@dedemorton the idea behind this is to make a clear reference from what a user see - an agent in the upgrade status after a stack update. 

There is no idea for the user that this is only a ghost entry and you need to unenroll this. So from a user perspective you will look at the troubleshooting section to see what you can do about an agent that is just showing `updating`. 

As this is relativly new I do not see myself in the position to describe the circumstances why this can happen. I have just taken the opportunity from the case I had with that situation to improve the documentation based on what I see during daily work.